backend/services/meetingHistory.js

```diff
@@ -23,8 +23,9 @@ module.exports = async function getMeetingHistory() {
             meetingType: page.properties.MeetingType.title[0].text.content,
             meetingNumber: page.properties['Meeting #'].number,
             numAttended: page.properties['Attended'].number,
-            numAttended: page.properties['# Unexcused Absences'].number,
-            attendancePercent: page.properties['%'].formula.number
+            numAbsent: page.properties['# Unexcused Absences'].number,
```

Comment:
Nit: not sure if using # or % would be a good idea here from a style perspective, we generally use those symbols for other things (ie comments or string formatting)